src/enums.h

```diff
@@ -560,12 +560,12 @@ struct ShopInfo
 {
 	uint16_t itemId = 0;
 	int32_t subType = 1;
-	int64_t buyPrice = 0;
-	int64_t sellPrice = 0;
+	uint32_t buyPrice = 0;
+	uint32_t sellPrice = 0;
```

Comment:
Why not uint64_t? Didn't the client start accepting larger numbers recently?

Reply:
Values currently allowed in version 13.x do not exceed the uint32_t limits.